src/Raven.Studio/typescript/viewmodels/database/query/query.ts

```diff
@@ -313,13 +314,15 @@ class query extends viewModelBase {
             this.queriedIndex() ? appUrl.forVisualizer(this.activeDatabase(), this.queriedIndex()) : null);
 
         this.isMapReduceIndex = ko.pureComputed(() => {
-            const currentIndex = this.getCurrentIndex();
+            this.getQueriedIndexInfo();
```

Comment:
we should call methods with side effects inside pure computed.

Reply:
done, moved to runQuery